### ISSUE TITLE
importError: DLL load failed(import h5py)

### ISSUE THREAD

**Issue by KyungdongKim** [h5py/h5py#907]
Hello.

I installed the OSGeo4W package to use h5py.

I used OSGeo shell to install h5py using pip install.  

<OSGeo4W shell>
"C:\Users\Public\Desktop\OSGeo4W>pip install h5py"

when I open QGIS and go to the Python Console in QGIS I get this error:
```
<QGIS2.18.10 - Python console>
>>>import h5py
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:/OSGEO4~1/apps/qgis/./python\qgis\utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "C:\OSGEO4~1\apps\Python27\lib\site-packages\h5py\__init__.py", line 43, in <module>
    from . import h5a, h5d, h5ds, h5f, h5fd, h5g, h5r, h5s, h5t, h5p, h5z
  File "C:/OSGEO4~1/apps/qgis/./python\qgis\utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "h5py\h5p.pxd", line 16, in init h5py.h5a (D:\Build\h5py\h5py-2.7.0\h5py\h5a.c:6968)
  File "C:/OSGEO4~1/apps/qgis/./python\qgis\utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ImportError: DLL load failed: ÁöÁ¤µÈ ÇÁ·Î½ÃÀú¸¦ Ã£À» ¼ö ¾ø½À´Ï´Ù.
```
i don't know how can use h5py in QGIS python console

please

Any help How to solve this problem 

Thanks

KKD

**Comment by tacaswell** [comment]
This might be fixed by #841, can you try installing from source?

**Comment by tacaswell** [comment]
Have you reported this to QGIS?   This looks like this traceback is entirely in their code.

**Comment by tacaswell** [comment]
xref #828

**Comment by tacaswell** [comment]
Closing as fixed by down-stream packaging, same as #828
